<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorDialog</class>
 <widget class="QDialog" name="MotorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MotorDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMotorX">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMotorY">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSyringe2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Syringe 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSyringe1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Syringe 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbResult">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Result</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leResult">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
